db/seeds.rb

```diff
@@ -6,3 +6,23 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Teacher.delete_all
+
+    teachers = Teacher.create!(
+                            [{
+        first_name: Faker::Name.first_name,
+        last_name: Faker::Name.last_name,
+        email: 'random@example.com', 
+        password: 'dupablada'},
+                               {
+        first_name: Faker::Name.first_name,
+        last_name: Faker::Name.last_name,
+        email: 'random1@example.com', 
+        password: 'dupablada'},
+                               {
+        first_name: Faker::Name.first_name,
+        last_name: Faker::Name.last_name,
+        email: 'random3@example.com', 
+        password: 'dupablada'}],
+    )
```

Comment:
Hey, I appreciate choosing my favourite style 😄 I hope you chose it because it makes sense to you too :) Just in case, let me suggest even more beautiful way of structuring the content(maybe `rubocop -a` would do that for you? 🤔 )

```suggestion
    teachers = Teacher.create!([
      {
        first_name: Faker::Name.first_name,
        last_name: Faker::Name.last_name,
        email: 'random@example.com', 
        password: 'dupablada'
      },
      {
        first_name: Faker::Name.first_name,
        last_name: Faker::Name.last_name,
        email: 'random1@example.com', 
        password: 'dupablada'
      },
      {
        first_name: Faker::Name.first_name,
        last_name: Faker::Name.last_name,
        email: 'random3@example.com', 
        password: 'dupablada'
      }
     ])
```